src/tests/Microsoft.Agents.Authentication.Msal.Tests/MSALHttpClientFactoryTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Moq;
+using System;
+using System.Net.Http;
+using Xunit;
+
+namespace Microsoft.Agents.Authentication.Msal.Tests
+{
+    public class MSALHttpClientFactoryTests
+    {
+        private readonly Mock<IServiceProvider> _service = new Mock<IServiceProvider>();
```

Comment:
we could use = new(); instead of new Mock<IServiceProvider>();

Reply:
Yes, I was avoiding that nomenclature, I don't like it 🙃

Reply:
Applied. Thanks!